.dep-versions

```diff
@@ -8,11 +8,11 @@ enzyme=v0.0.130
 
 # For a custom PL version, update the package version here and at
 # 'doc/requirements.txt
-pennylane=0.38.0
+pennylane=0.39.0.dev15
 
 # For a custom LQ/LK version, update the package version here and at
 # 'doc/requirements.txt'. Also, update the 'LIGHTNING_GIT_TAG' at
 # 'runtime/Makefile' and at all GitHub workflows, using the exact
 # commit hash corresponding to the merged PR that implements the
```

Comment:
Please make sure to also update the `LIGHTNING_GIT_TAG` variable as described in this comment :)

Reply:
Could you provide some more details on this? This is my first Catalyst contribution and I'm not sure I understand the comment :confused: